pkg/subscription/list.go

```diff
@@ -0,0 +1,40 @@
+package subscription
+
+import (
+	"fmt"
+
+	profilesv1 "github.com/weaveworks/profiles/api/v1alpha1"
+)
+
+// SubscriptionDescription contains a description of a subscription
+type SubscriptionDescription struct {
```

Comment:
this name cracks me up

Reply:
actually this is very similar to the `get` one.. any way we can use one and polulate/extract things we need for the display? i think we do this for `search` vs `show`